nvme-print.h

```diff
@@ -26,6 +26,7 @@ struct print_ops {
 	void (*phy_rx_eom_log)(struct nvme_phy_rx_eom_log *log, __u16 controller);
 	void (*ctrl_list)(struct nvme_ctrl_list *ctrl_list);
 	void (*ctrl_registers)(void *bar, bool fabrics);
+	void (*ctrl_register)(int offset, uint64_t value);
```

Comment:
So first, do the nvme-print.c changes and then update the stdout and json output accordingly.